apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -272,42 +278,35 @@ const Portfolio = ({ selectedWalletOption }: PortfolioProps) => {
   const showSmallBalances = useShowSmallBalances();
 
   const portfolioBalances = React.useMemo<BalanceItem[]>(() => {
-    const tokenBalances = Object.values(allBalances).reduce<Record<string, BalanceItem>>(
+    const balanceTokens = Object.values(allBalances).reduce<Record<string, BalanceToken>>(
       (acc, { balancesAndPrices, isLoadingChainPrices }) => {
         Object.entries(balancesAndPrices).forEach(([tokenAddress, tokenInfo]) => {
           const tokenKey = `${tokenInfo.token.chainId}-${tokenAddress}`;
           // eslint-disable-next-line no-param-reassign
           acc[tokenKey] = {
-            tokens: [
-              {
-                balance: 0n,
-                token: tokenInfo.token,
-                balanceUsd: 0,
-                price: tokenInfo.price,
-              },
-            ],
-            totalBalanceInUnits: '0',
-            totalBalanceUsd: 0,
+            balance: 0n,
+            token: tokenInfo.token,
+            balanceUsd: 0,
+            price: tokenInfo.price,
             isLoadingPrice: isLoadingChainPrices,
-            relativeBalance: 0,
           };
 
           Object.entries(tokenInfo.balances).forEach(([walletAddress, balance]) => {
-            const tokenBalance = acc[tokenKey].tokens[0].balance + balance;
+            const tokenBalance = acc[tokenKey].balance + balance;
 
             if (selectedWalletOption === ALL_WALLETS) {
```

Comment:
this could just be `selectedWalletOption === ALL_WALLETS || selectedWalletOption === walletAddress` right?